studentList.js

```diff
@@ -1,17 +1,18 @@
 export const students = [
-	{ name: 'Sofi Baazov', photo: null, bio: null, cvLink: null },
-	{ name: 'Anna Fridman', photo: null, bio: null, cvLink: null },
-	{ name: 'Deadrop Furman', photo: null, bio: null, cvLink: null },
-	{ name: 'Leonid Garchu', photo: null, bio: null, cvLink: null },
-	{ name: 'Shay Geva', photo: null, bio: null, cvLink: null },
-	{ name: 'Uri Koffman', photo: null, bio: null, cvLink: null },
-	{ name: 'Effi Malach', photo: null, bio: null, cvLink: null },
-	{ name: 'Haitham Mansour', photo: null, bio: null, cvLink: null },
-	{ name: 'Yakov Markman', photo: null, bio: null, cvLink: null },
-	{ name: 'Kobi Mechani', photo: null, bio: null, cvLink: null },
-	{ name: 'Omer Messing', photo: null, bio: null, cvLink: null },
-	{ name: 'Nivo music', photo: null, bio: null, cvLink: null },
-	{ name: 'Barak Ostroumov', photo: null, bio: null, cvLink: null },
-	{ name: 'Areen Shalli', photo: null, bio: null, cvLink: null },
-	{ name: 'Tal Sprung', photo: null, bio: null, cvLink: null },
+	{ name: 'Sofi Baazov', photo: "https://oldough.com/static/images/avatar.png", bio: null, cvLink: null },
```

Comment:
if this is a generic avatar, it's better to put it in the render function in case the photo is `null`